turbopuffer/backend.py

```diff
@@ -116,9 +116,16 @@ def make_api_request(self,
                     match_processing = re.match(r'.*processing_time;dur=([\d\.]+)', server_timing_str)
                     if match_processing:
                         try:
+                            # todo: update this to `processing_time`?
                             performance['server_time'] = float(match_processing.group(1)) / 1000.0
```

Comment:
left this as `server_time` instead of changing to `processing_time` for backwards compatibility. Probably worth updating when we do a semver release with a MAJOR version update